app/lib/activitypub/activity/create.rb

```diff
@@ -185,6 +185,43 @@ def process_emoji(tag)
     emoji.save
   end
 
+  def process_converted_media
+    as_array(@object['url']).each do |attachment|
+      if attachment.is_a?(String)
+        media_type = nil
+        size       = nil
+        href       = attachment
+        blurhash   = nil
+        focus      = nil
+      else
+        media_type = attachment['mediaType']
+        size       = attachment['size']&.to_i
```

Comment:
Note that `size` is definitely not standard, but it is what Peertube uses, and it lets us ignore bigger media easily